### PR TITLE
Mark attendance on login.

### PR DESCRIPTION
**Pull request by shridarpatil**
Activate checkin / checkout in attendance control panel / HR settings.
On login, if not checked in, show popup to allow checkin

Closed PR: https://github.com/frappe/erpnext/pull/11054
issue: https://github.com/frappe/erpnext/issues/428

### CODE REVIEW

**Comment by shridarpatil** [comment]
I think the build is getting failed because of this patch https://github.com/frappe/erpnext/blob/develop/erpnext/patches.txt#L492


image

File: erpnext/hr/doctype/attendance/attendance.py [hunk 1]
```diff
@@ -54,3 +54,61 @@ def validate(self):
 		self.validate_attendance_date()
 		self.validate_duplicate_record()
 		self.check_leave_record()
+
+
+def employee():
```
Comment (nabinhait) [inline erpnext/hr/doctype/attendance/attendance.py]: `employee` is not a good method name, `get_employee()` ???

File: erpnext/public/js/checkin_checkout_attendance.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+						show_popup();
+					}
+				}
+			});
+		}
+	});
+
+
+function show_popup() {
+	var dialog = new frappe.ui.Dialog({
```
Comment (nabinhait) [inline erpnext/public/js/checkin_checkout_attendance.js]: If user does not logged out till the next date, what will happen? May be check the server time every few minutes and if day changes, show the popup again.

**Review by nabinhait — CHANGES_REQUESTED** [review]
- Add few screenshots
- Add documentation

**Comment by nabinhait** [comment]
Closing this for now, please reopen after fixing.

**Pull request closed without merging**